### ISSUE TITLE
Example app crashes 

### ISSUE THREAD

**Issue by shanehartman** [xmartlabs/XLActionController#70]
Example app crashes in this method in DynamicCollectionViewFlowLayout:

 override open func layoutAttributesForElements(in rect: CGRect) -> [UICollectionViewLayoutAttributes]? {
        guard let animator = dynamicAnimator else {
            return super.layoutAttributesForElements(in: rect)
        }
        
        return animator.items(in: rect) as? [UICollectionViewLayoutAttributes]
    }

Here's the console log:
fatal error: NSArray element failed to match the Swift Array Element type
2017-10-23 16:27:45.074075-0400 Example[18698:29187926] fatal error: NSArray element failed to match the Swift Array Element type
(lldb) 

Using XCode 9 with Swift 4. Choose Tweetbot from the tableview sample app options.

**Comment by hammad** [comment]
I am also getting the same crash. I think this started to happen after iOS 11 release.

**Comment by xs6615** [comment]
Getting same crash too.  Both in the example app and inside my own app with XLActionController setup with cocoa pods

**Comment by msalihkocak** [comment]
I am also getting the same crash in my app released on App Store. It caused more than 500 crashes and users writing down bad comments about it.

**Comment by fadylateef** [comment]
Any solutions yet ! 
This problem shown on iOS 11 Devices.

**Comment by HyunjoonKo** [comment]
I have modified the code in DynamicCollectionViewFlowLayout.swift as follows: 

```
override open func layoutAttributesForElements(in rect: CGRect) -> [UICollectionViewLayoutAttributes]? {
        guard let animator = dynamicAnimator else {
            return super.layoutAttributesForElements(in: rect)
        }
        
        if let items = animator.items(in: rect) as? [AnyObject] {
            return items.filter{ $0 is UICollectionViewLayoutAttributes }.map{ $0 as! UICollectionViewLayoutAttributes }
        }

        return nil
        //return animator.items(in: rect) as? [UICollectionViewLayoutAttributes]
}
```

Fortunately, this code works well.

**Comment by m-revetria** [comment]
Hi guys, sorry for the delay to respond.

This is fixed on #76.

**Comment by m-revetria** [comment]
#76 was just merged
